tests/bears/requirements/PipRequirementTest.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import shutil
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+@unittest.skipIf(shutil.which('pip') is None, "Pip is not installed.")
+class PipRequirementTestCase(unittest.TestCase):
+
+    def test_Install_Command_With_Version(self):
```

Comment:
we usually use snake case for functions, that means all lower case and separated by underscores - can you make that right? :)